library/public/options.h.in

```diff
@@ -108,7 +108,7 @@ public:
 
   /**
    * Templated parsing method used internally to parse strings.
-   * Implemented only for: bool, int, double, ratio_t, string and double vector.
+   * Implemented only for: bool, int, double, ratio_t, int, double and string vector.
```

Comment:
doc not clear, maybe list individual types and specify that vectors encapsuling them is also supported.